docs/credentials/credential-sharing.md

```diff
@@ -11,20 +11,21 @@ contentType: howto
 Available on all Cloud plans, and Enterprise self-hosted plans.
 ///
 
-You can share a credential you created with other users in the same n8n workspace as you. The other users can then use the credential in their workflows. They can't access or edit the credential details.
+You can share a credential directly with other users to use in their workflows. Or into a project for all members of a project to use in workflows inside that project. Any users accessing a shared credential won't be able to access or edit the credential details.
```

Comment:
Reading again here, have some minor tweaks to this sentence to update it as follows:

"You can share a credential directly with other users to use in their own workflows. Or share a credential into a project for all members of that project to use. Any users using a shared credential won't be able to view or edit the credential details."